components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Runtime/PublishServicesConfig.cs

```diff
@@ -42,7 +50,7 @@ public class PublishServicesConfig : ConfigBase, IPublishServicesConfig {
                     () => _defaultBatchTriggerInterval));
 
                 if (batchInterval.TotalMilliseconds >= 100 && batchInterval.TotalMilliseconds <= 3600000) {
-                    //ToDo: This behavior should be logged.
+                    _logger.Verbose($"DefaultBatchTriggerInterval set to {batchInterval}");
```

Comment:
@karok2m I added logging to this file. Is this what you had in mind? Verbose, Debug or Information?

Reply:
Ok, so the warning should be shown inside of "else" and not in the "if". I simplified the msg as follows as it makes managing the ranges easier: "DefaultBatchTriggerInterval: Provided value {batchInterval} should be >= 100 and <= 3600000. Defaulting to {_defaultBatchTriggerInterval.TotalMilliseconds}.".
Let me know what you think :)

Reply:
Message looks good.